lang/test/org/partiql/lang/eval/EvaluatingCompilerIntTest.kt

```diff
@@ -251,8 +251,9 @@ class EvaluatingCompilerIntTest : EvaluatorTestBase() {
         expectedPermissiveModeResult = "MISSING"
     )
 
-    @Test
```

Comment:
Was this test previously failing in this PR? Why is it being commented out in this PR?

Reply:
It works previously, since the default `LEGACY` mode `CAST` supports evaluating extremely small numbers. However, in the `HONOR_PARAMETERS` mode, it throws error when processing extremely small numbers. 

As what I tested, PostgreSQL does not allow processing extremely small numbers. I don't know if we want to support it or not. So I created another issue here, https://github.com/partiql/partiql-lang-kotlin/issues/784